.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "panda"]
   path = panda
-  url = ../../commaai/panda.git
+  url = ../../jnewb1/panda.git
```

Comment:
merged the other PRs, can you revert this?

Reply:
done